MultiPage-Clone/css/main.css

```diff
@@ -1,3 +1,528 @@
+
+header {
+  position: absolute;
+  top: 0px;
+  width: 100%;
+  height: 100px;
+  background-color: var(--White);
+}
+.header-content {
+  height: 100%;
+  margin: 0px 30px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+.for-header {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: calc(var(--gap) / (3 / 2));
+}
+.inside-header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: var(--gap);
+}
+
+.logo {
+  width: 20px;
+  height: 20px;
+  background-color: var(--Royal-blue);
+}
+
+.for-name {
+  text-decoration: none;
+  color: var(--Black);
+  cursor: pointer;
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 1em;
+}
+
+.for-title {
+  text-decoration: none;
+  color: var(--Black);
+  cursor: pointer;
+  font-family: "Poppins", sans-serif;
+  font-weight: 300;
+  font-size: 20px;
+  line-height: 1em;
+  width: fit-content;
+}
+
+/* Hamburger menu */
+
+.hamburger-cont {
+  width: 100vw;
+  min-height: 0px;
+  position: absolute;
+  top: 0px;
+  left: 0px;
+}
+
+.hamburger-menu {
+  display: flex;
+  flex-direction: column;
+  gap: var(--hamburger-gap);
+  width: max-content;
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  margin: var(--hamburger-margin);
+  z-index: 3;
+  cursor: pointer;
+}
+.hamburger-menu::before,
+.hamburger-menu::after,
+.hamburger-menu input {
+  content: "";
+  width: var(--bar-width);
+  height: var(--bar-height);
+  background-color: var(--Royal-blue);
+  border-radius: 9999px;
+  transition: transform var(--animation-timing),
+    background-color var(--animation-timing);
+}
+.hamburger-menu::before {
+  width: calc(var(--bar-width) / 2);
+}
+.hamburger-menu:has(input:checked)::before {
+  transform-origin: right center;
+  transform: translate(-71%, -63%) rotate(-135deg);
+  background-color: var(--Black);
+}
+.hamburger-menu::after {
+  width: calc(var(--bar-width) / 2);
+  align-self: flex-end;
+}
+.hamburger-menu:has(input:checked)::after {
+  transform-origin: left center;
+  transform: translate(70%, 63%) rotate(-135deg);
+  background-color: var(--Black);
+}
+.hamburger-menu input:checked {
+  transform: rotate(-45deg);
+  background-color: var(--Black);
+}
+
+.hamburger-menu input {
+  appearance: none;
+  outline: none;
+  pointer-events: none;
+}
+
+.sidebar {
+  min-width: 100vw;
+  min-height: 100vh;
+  background-color: var(--White);
+  position: absolute;
+  z-index: 2;
+  padding-top: 90px;
+  display: none;
+}
+
+.hamburger-menu:has(input:checked) + .sidebar {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: var(--Light-beige);
+  opacity: 1;
+}
+.menu-nav {
+  padding-top: 30px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 23px;
+}
+
+.menu-nav .nav-links1 {
+  color: var(--Royal-blue);
+}
+
+.nav-links {
+  text-decoration: none;
```

Comment:
![image](https://github.com/CodeYourFuture/Module-HTML-CSS/assets/118762745/f965b190-3b2f-404b-80de-770486ea49ce)
 When i hover over your links they go BOLD. The deom does not do this:
 
![image](https://github.com/CodeYourFuture/Module-HTML-CSS/assets/118762745/7e345c85-64b7-4942-8442-19801b340ec6)